grails-app/controllers/org/pih/warehouse/inventory/StockMovementController.groovy

```diff
@@ -211,12 +211,17 @@ class StockMovementController {
                 try {
                     stockMovementService.deleteStockMovement(params.id)
                     flash.message = "Successfully deleted stock movement with ID ${params.id}"
+                    params.deleted = 'true'
```

Comment:
This is not a change request but rather a discussion point.

do you think that maybe it would be better instead of passing this `deleted` flag to pass

```groovy
params.notificationMessage = g.message(
'react.stockMovement.deleted.success.message.label',
'Stock Movement has been deleted successfully',
)
params.notificationType='SUCCESS'

```

because at the moment your `notification-utils` are very specific to the case of deleting a stock Movement and if we ever want to add an exact same logic for example a success message of invoice or something like that we would need to add an additional utils function for that specific case

Reply:
I think it is a good idea. I am just worry about the length of the params. If there is nothing against passing this kind of param, I will do it.

Reply:
if we were about to go this way, I'd consider adding an enum for the notification type on the backend side similar to the one on the frontend side. I don't think you have to worry about "the length the param", because "inside" it will be only a simple string after the g.message executes.

Reply:
The max url length is 2048 characters, so on that part it should be ok.
Also just as I mentioned in one of my comments you could clear that param once you display the notification